route_planning_msgs/msg/RegulatoryElement.msg

```diff
@@ -4,15 +4,18 @@ uint8 TYPE_STOP=2
 uint8 TYPE_TRAFFIC_LIGHT=3
 uint8 TYPE_RIGHT_OF_WAY=4
 uint8 TYPE_SPEED_LIMIT=5
-uint8 type # Type of the regulatory element
+uint8 type
 
-uint8 STATE_UNKNOWN=0 # Unknown value
-uint8 MOVEMENT_ALLOWED=1 # Movement allowed for stop and yield-signs as well as traffic lights
-uint8 MOVEMENT_RESTRICTED=2 # Movement restricted for stop and yield-signs as well as traffic lights
-uint8 SPEED_30=30 # Speed value [km/h] for speed limit elements
-uint8 SPEED_50=50 # Speed value [km/h] for speed limit elements
-uint8 SPEED_70=70 # Speed value [km/h] for speed limit elements
-uint8 value # Signal value of the regulatory element (dependant on type)
+uint8 META_VALUE_STATE_UNKNOWN=0
+# meta values for YIELD, STOP, TRAFFIC_LIGHT
+uint8 META_VALUE_MOVEMENT_ALLOWED=1     # movement is allowed
+uint8 META_VALUE_MOVEMENT_RESTRICTED=2  # movement is restricted
+# meta values for SPEED_LIMIT
+uint8 META_VALUE_SPEED_1=1              # speed limit of 1km/h
+uint8 META_VALUE_SPEED_30=30            # speed limit of 30km/h
+uint8 META_VALUE_SPEED_50=50            # speed limit of 50km/h
+uint8 META_VALUE_SPEED_70=70            # speed limit of 70km/h
+uint8 meta_value
 
-geometry_msgs/Point[2] effect_line # effect line of the respective regulatory element, restricting the movement (stop, yield, traffic-light) or indicating the validity of a speed limit; z>=0 indicates the s-coordinate of the ref-path of effect-line-intersection (z=-1, if not part of ref-path)
-geometry_msgs/Point[] signal_positions # position(s) of the actual signal (e.g. position of the traffic-light or sign)
\ No newline at end of file
+geometry_msgs/Point[2] effect_line      # effect line of regulatory element, e.g., stop line
```

Comment:
What about [cancel lines](https://github.com/fzi-forschungszentrum-informatik/Lanelet2/blob/master/lanelet2_core/doc/RegulatoryElementTagging.md#parameters)?

Reply:
I would say, that cancle lines are rather unusual and can also be implemented as a new regulatory element.